src/api/category.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+
+const CategoryUrl = '/api/categories';
+
+
+
```

Comment:
why is there so much space between codes? it affects the readability, you can reduce this